gatsby-config.js

```diff
@@ -31,19 +31,9 @@ module.exports = {
       },
     },
     {
-      resolve: `gatsby-source-buttercms`,
+      resolve: require.resolve(`gatsby-source-buttercms`),
```

Comment:
Is there a reason this uses a local version of the source plugin, rather than installing from npm?

Reply:
seems like using a local version will prevent people using the starter from opting in to updates to the plugin in the future

Reply:
Thanks for the feedback! It totally makes sense. However this one: `require.resolve('gatsby-source-buttercms')` was also taking plugin from `node_modules`. Anyway, using `require.resolve()` isn't necessary here.

Reply:
Fixed now ✅

Reply:
gotcha!